### PR TITLE
New blog post

### PR DESCRIPTION
**Pull request by odersky**

### CODE REVIEW

File: blog/_posts/2016-02-17-scaling-dot-soundness.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+layout: blog
+post-type: blog
+by: Martin Odersky
+title: Scaling DOT to Scala: Soundness
+disqus: true
+---
+
+In my [last
+blog](http://www.scala-lang.org/blog/2016/02/03/essence-of-scala.html)
```
Comment (sjrd) [inline blog/_posts/2016-02-17-scaling-dot-soundness.md]: last blog <ins>post</ins>

File: blog/_posts/2016-02-17-scaling-dot-soundness.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+I introduced DOT, a minimal calculus that underlies much of Scala.
+DOT is much more than an academic exercise, because it gives us
+guidelines how to design a sound type system for full Scala. As was
```
Comment (sjrd) [inline blog/_posts/2016-02-17-scaling-dot-soundness.md]: guidelines <ins>on</ins> how?

**Comment by sjrd** [comment]
That's all for me.

**Comment by densh** [comment]
The published blog post page lacks usual formatting / looks like plain text: http://scala-lang.org/blog/2016/02/17/scaling-dot-soundness.html

**Pull request merged**